apluslms_roman/backends/docker.py

```diff
@@ -17,6 +17,7 @@
 
 logger = logging.getLogger(__name__)
 
+LABEL_PREFIX = 'io.github.apluslm.roman'
```

Comment:
This could be here: https://github.com/apluslms/roman/blob/a8df0dc2324baa6d30289b2feb44e5c9349eb622/apluslms_roman/backends/__init__.py#L98

Reply:
You left it here too...